### PR TITLE
doc: add GetRoutes method

### PR DESCRIPTION
**Pull request by li-jin-gou**
refer to https://github.com/gofiber/fiber/pull/2112

### CODE REVIEW

File: api/app.md [hunk 1]
```diff
@@ -500,6 +500,41 @@ func main() {
 ```
 {% endcode %}
 
+## GetRoutes
+
+This method gets get all routes, when filterUse equal to true. it will filter the routes registered by the middleware.
```
Comment (gaby) [inline api/app.md:505]: The first part of the sentence should be "This method gets all routes".

**Review by gaby — CHANGES_REQUESTED** [review]

File: api/app.md (hunk 1 above)
Comment (li-jin-gou) [reply api/app.md:505]: thanks

Comment (li-jin-gou) [reply api/app.md]: I adjusted the document

File: api/app.md [hunk 2]
```diff
@@ -500,6 +500,42 @@ func main() {
 ```
 {% endcode %}
 
+## GetRoutes
+
+This method gets get all routes.
```
Comment (gaby) [inline api/app.md:505]: Should this one still say "Gets get" ?

**Review by gaby — CHANGES_REQUESTED** [review]

File: api/app.md (hunk 2 above)
Comment (li-jin-gou) [reply api/app.md]: done

**Review by gaby — APPROVED** [review]

File: api/app.md (hunk 2 above)
Comment (gaby) [reply api/app.md]: Looks good

**Pull request merged**